hw/top_earlgrey/data/ip/chip_aes_testplan.hjson

```diff
@@ -0,0 +1,234 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_aes
+  testpoints: [
+    {
+      name: chip_sw_aes_enc
+      desc: '''Verify the AES operation.
+
+            Write a 32-byte key and a 16-byte plain text to the AES registers and trigger the AES
+            computation to start. Wait for the AES operation to complete by polling the status
+            register. Check the digest registers for correctness against the expected digest value.
+            Repeat the test for all three supported key lengths (128/192/256).
+            '''
+      features: [
+        "AES.MODE.ECB",
+      ]
+      stage: V2
+      si_stage: SV1
+      lc_states: ["TEST_UNLOCKED", "DEV", "PROD", "PROD_END", "RMA"]
+      tests: ["chip_sw_aes_enc",
+              "chip_sw_aes_enc_jitter_en"]
+    }
+    {
+      name: chip_sw_aes_multi_block
+      desc: '''Verify the AES operations on multi-block messages.
+            This test should be performed for all supported modes (ECB, CBC, CFB-128, OFB and CTR)
+            and all supported key lengths (128/192/256).
+
+            Note: The number of message blocks can be arbitrarily chosen (at least 2) or randomized
+
+            Note: AES should start this test using the automatic (default) operation. This means
+                  that the MANUAL_OPERATION bit in CTRL_SHADOWED register is set to 0.
+
+            Procedure:
+            -AES should be idle at the begining. Check this by reading the STATUS.IDLE bit.
+            -Set the mode by writing to CTRL_SHADOWED.MODE
+            -Write CTRL_SHADOWED.KEY_LEN to set the key length.
+            -Write to CTRL_SHADOWED.OPERATION to choose encryption or decryption.
+            -Write a key to KEY_SHARE registers
+            -If testing CBC or CTR modes, write the initialization vector to IV registers. Each IV
```

Comment:
You also need to provide an IV for CFB and OFB modes by the way. But also here, you can make your life easier by referring to the corresponding DIF functions.

Reply:
OK. I've changed it now